docs/jsonapi-spec.md

```diff
@@ -473,10 +473,38 @@ Commands are defined using the BNF-like syntax, with samples presented using a [
 
 ### countDocuments Command
 
+Returns the count of documents that match the query for a collection or view.
+
+*Sample:*
+
+```json
+db.orders.countDocuments({})
```

Comment:
The json needs to be like `{
                        "countDocuments": {
                            "filter": {"location": "London", "race.competitors" : {"$eq" : 100}}
                        }
                      }`

Reply:
Thanks!